### PR TITLE
[velero] Bump velero version v1.11.0

### PR DESCRIPTION
**Pull request by qiuming-best**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the values.yaml or README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

**Review by jenting — APPROVED** [review]
LGTM

File: charts/velero/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: 1.10.2
+appVersion: 1.11.0
```
Comment (ywk253100) [inline charts/velero/Chart.yaml:2]: Don't we need to update the image version somewhere?

File: charts/velero/values.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 # enabling node-agent). Required.
 image:
   repository: velero/velero
-  tag: v1.10.2
```
Comment (qiuming-best) [inline charts/velero/values.yaml:9]: @ywk253100 thanks for your checking, I've updated

**Review by jenting — COMMENTED** [review]
@qiuming-best  Should we need to bump the plug-ins version as well?

**Review by ywk253100 — DISMISSED** [review]

File: charts/velero/values.yaml [hunk 3]
```diff
@@ -53,13 +53,13 @@ dnsPolicy: ClusterFirst
 # If the value is a string then it is evaluated as a template.
 initContainers:
   # - name: velero-plugin-for-csi
-  #   image: velero/velero-plugin-for-csi:v0.4.2
+  #   image: velero/velero-plugin-for-csi:v0.5.0
```
Comment (qiuming-best) [inline charts/velero/values.yaml:56]: @jenting Also I've update the version of plugins

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**